woocommerce/woocommerce-gutenberg-products-block
# Products by Rating: Improve Rating filter accessibility

This PR utilizes `speak` to announce a Rating filter change to screen readers.

#### Accessibility




### Testing
1. Check out this PR, build.
2. Add a new page, add All Products and the Filter by Ratings block.
3. Use a screen reader
4. Try ticking and unticking the Ratings filter checkboxes
5. Test with both the Apply filters button enabled and disabled
6. Make sure the screen reader announces 'Rated %s filter added' and 'Rated %s filter removed.'

#### Automated Tests

#### User Facing Testing

Same as the above testing instructions.


### WooCommerce Visibility




### Changelog

> Filter by Ratings: Improve accessibility by announcing the rating filter change
